### DISCUSSION TITLE
Release 3.1

### DISCUSSION THREAD

**Discussion by lukevp** [lukevp/ESC-POS-.NET#261] [General]
@igorocampos  we should discuss preparing a 3.1 release with the updated encoding information and barcode changes (pending the merge), or a 4.0 release if there are any other major changes that you think we should make.  One of the things that was recently brought up is that the status reading code no longer works in 3.0.  Is there anything open that we should incorporate into a new release?

**Comment by igorocampos** [comment]
Yes, I've just submit #262 as mentioned in the Encoding PR. We could also include that in the next release. And there's a question there for you, regarding a breaking change. If you prefer doing the breaking change, then I'd recommend going to 4.0 instead of 3.1

About the status reading code not working, I have no clue about that, and no printer to test/propose a fix...
Text printing is something I test using a text file or such, but I don't think I could fully test any status reading code without a printer.

**Comment by igorocampos** [comment]
There's also pretty much ready PRs: #217, #234, #196, #195, #165
Just pending a decision from you, or little adjustments.

And I don't know if you have already addressed #125, #143 and #130  elsewhere, but seems important.
